lib/teracy-dev-essential/config/host_manager.rb

```diff
@@ -36,11 +49,70 @@ def configure_node(settings, config)
         config.vm.provision "guest-hosts-fixer", type: "shell" do |provision|
           provision.set_options(options)
         end
-
       end
 
       private
 
+      def get_aliases plugins
+        plugin = plugins.find { |p| p['name'] == PLUGIN_NAME }
+
+        return [] if plugin.nil?
+
+        if plugin['options'] && plugin['options']['aliases']
+          plugin_aliases = plugin['options']['aliases']
+        end
+
+        plugin_aliases ||= []
+      end
+
+      def get_etc_hosts
+        return @etc_hosts unless @etc_hosts.nil?
+
+        if Vagrant::Util::Platform.windows?
+          @etc_hosts = `type C:\\Windows\\System32\\drivers\\etc\\hosts`
+        else
+          @etc_hosts = `cat /etc/hosts`
+        end
+
+        if !$?.success?
+          return nil
+        end
+
+        @etc_hosts
+      end
+
+      def check_conflict_hostname aliases
+        @logger.debug('check conflict hostname')
+
+        etc_hosts = get_etc_hosts
+
+        if etc_hosts.nil?
+          @logger.warn('Reading /etc/hosts with no success, aborted')
+
+          return false
+        end
+
+        @logger.debug("etc_hosts: #{etc_hosts}")
+
+        conflict_list = []
+        
+        hex = "[0-9a-fA-F]+"
+        sign = "\.?\:?"
+        ip = "#{hex}#{sign}#{hex}#{sign}#{hex}#{sign}#{hex}#{sign}#{hex}#{sign}#{hex}#{sign}#{hex}#{sign}#{hex}"
```

Comment:
Could you give example of which kind of ip you looking for with this regex? IPV4 IPV6.
For me it look like `10.0.0.13` doesn't match this regex

Reply:
thanks for the check, I've updated it, pls check again

Sample hosts file I'm looking for:

```


192.168.1.86	dev.iorad.local
192.168.1.86	node-01.local


127.0.0.1       dev.iorad.local

2405:4800:528f:ed7:fb:d787:6821:a68a dev.iorad.local

2405:4800:528f:ed7:c5c1:1dae:a7a:d94c node-01.local
```